layouts/New/Search.tsx

```diff
@@ -0,0 +1,16 @@
+'use client';
```

Comment:
Why is this a Client component? 🤔

Reply:
I need to use React hooks and it breaks on the server

Reply:
Right, but this component has only children that might use hooks. This component itself is doing nothing that requires "use client".

Next.js/React can have "client-side" components as children of server components. "use client" shouldn't be needed here. If it gives an error, the source of error is somewhere else.

Adding it here also fixes it, but again, the source is somewhere else.